source/_patterns/01-molecules/navigation/view-selector.mustache

```diff
@@ -1,11 +1,11 @@
-<div class="view-selector{{#figureUrl}} view-selector--has-figures{{/figureUrl}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
+<div class="view-selector{{#figureUrl}} view-selector--has-figures{{/figureUrl}}{{#isTabSelector}} button--switch-selector{{/isTabSelector}}" data-behaviour="ViewSelector"{{#sideBySideUrl}} data-side-by-side-link="{{sideBySideUrl}}"{{/sideBySideUrl}}>
   <ul class="view-selector__list">
     <li class="view-selector__list-item view-selector__list-item--article{{^figureIsActive}} view-selector__list-item--active{{/figureIsActive}}">
-      <a href="{{articleUrl}}" class="view-selector__link view-selector__link--article"><span>Article</span></a>
+      <a href="{{articleUrl}}" class="view-selector__link view-selector__link--article{{#isTabSelector}} primary-selector{{/isTabSelector}}"><span>Article</span></a>
```

Comment:
Please remove the spans

Reply:
It is done in another PR - https://github.com/elifesciences/pattern-library/pull/1199